app/javascript/packs/show_image.js

```diff
@@ -0,0 +1,18 @@
+$('#input').on('change', function() {
+    showImage(this);
+})
+
+function showImage(input) {
```

Comment:
again, arrow functions, showImage is good enough, I can understand what this function do

Reply:
arrow functions haven't "this", if I understand MDN clearly, so can I leave line $('#input').change(function () {showImage(this)}) as it is?

Reply:
but you're not using 'this' inside showImage function, you're passing it as parameter, arrow functions didn't have their own 'this' context, but you can work with parameters

Reply:
yes you can leave it in this case, but don't make it one-line